Library/Homebrew/cmd/tap.rb

```diff
@@ -52,7 +52,12 @@ class TapCmd < AbstractCommand
       sig { override.void }
       def run
         if args.repair?
-          Tap.installed.each do |tap|
+          taps = Tap.installed
+          taps &= args.named.to_taps if args.named.present?
+
+          taps.each do |tap|
+            puts "Checking if we need to repair #{tap.full_name}..." if args.verbose?
+            Tap.installed.include?(tap)
```

Comment:
This doesn't do anything.

Reply:
Thanks. I just noticed that too when reviewing the code coverage report. Line 60 shouldn't have been committed. Removed.